1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,7 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter(indexes);
```

Comment:
Please check this to find more about the filter method
https://www.jamesqquick.com/blog/javascript-array-filter-explained-in-5-minutes/
https://www.w3schools.com/jsref/jsref_filter.asp
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter